packages/styles/dialog.css

```diff
@@ -166,20 +177,14 @@
   color: var(--gray-20);
 }
 
-.cauldron--theme-dark .Dialog__heading:focus,
-.cauldron--theme-dark .Dialog__close:focus,
-.cauldron--theme-dark .Dialog__close:hover {
-  border-bottom-color: var(--white);
+.cauldron--theme-dark .Dialog__close:active {
+  color: var(--accent-medium);
 }
 
 .cauldron--theme-dark .Modal--info .Dialog__header,
 .cauldron--theme-dark .Modal--info .Dialog__content,
 .cauldron--theme-dark .Modal--info .Dialog__footer {
-  background-color: var(--accent-medium);
-}
-
-.cauldron--theme-dark .Modal--info .Dialog__header {
```

Comment:
I wonder if it is necessary to add the border-bottom for info Modal. At least the new ad modal I am looking at does not have that.

Reply:
This is what's in the pattern library at least for the "simple" variation: https://preview.uxpin.com/520e80315786a67ba43a3ca50598e6d69782627c#/pages/161871161/simulate/no-panels?mode=i

Reply:
I see. I guess we will have to remove the bottom border for multiple places in the extension.